justfile

```diff
@@ -66,6 +66,9 @@ fresh-start:
 	{{ RUN_WEB }} flask add-department "Seattle Police Department" "SPD" "WA"
 	{{ RUN_WEB }} flask bulk-add-officers /data/init_data.csv
 
+	# Stamp the database
+	@just db stamp
```

Comment:
This is purely pedantic but I think it might make more sense to add this right after the `python create_db.py` line

Reply:
Done in [c13a8cd](https://github.com/OrcaCollective/OpenOversight/pull/464/commits/c13a8cd958f8e21bb2e9370fad1d291fe4c465a8)!